imagebuildah/stage_executor.go

```diff
@@ -446,6 +500,63 @@ func (s *StageExecutor) runStageMountPoints(mountList []string) (map[string]inte
 					if fromErr != nil {
 						return nil, errors.Wrapf(fromErr, "unable to resolve argument %q", kv[1])
 					}
+					// If additional buildContext contains this
+					// give priority to that and break if additional
+					// is not an external image.
+					if additionalBuildContext, ok := s.executor.additionalBuildContexts[from]; ok {
+						if additionalBuildContext != nil {
```

Comment:
If we don't set `nil` values in the map, we don't need to check for them.

Reply:
Same just added to prevent any unintentional SEGSEV in future if someone edits a top level code.